internal/events/get.go

```diff
@@ -34,7 +34,7 @@ type flagsEvents struct {
 	End     uint64 `flag:"end" info:"End block height"`
 	Last    uint64 `default:"10" flag:"last" info:"Fetch number of blocks relative to the last block. Ignored if the start flag is set. Used as a default if no flags are provided"`
 	Workers int    `default:"10" flag:"workers" info:"Number of workers to use when fetching events in parallel"`
-	Batch   uint64 `default:"250" flag:"batch" info:"Number of blocks each worker will fetch"`
+	Batch   uint64 `default:"25" flag:"batch" info:"Number of blocks each worker will fetch"`
```

Comment:
I think the best is to use `backend.DefaultMaxHeightRange` so it directly gets the default value from the AN. Since it's not possible to do that in struct tag you should remove the default value from the tag and programmatically check if the variable has a null value and assign default.